app/views/content/landing/is-teaching-right-for-me/_a_job_like_no_other.html.erb

```diff
@@ -0,0 +1,32 @@
+<section class="container">
+  <div>
+    <h2 class="heading--box-blue">A job like no other</h2>
+    <p class="inset">
+      Looking for challenge and variety? As a teacher you'll work with lots of young people with a range of abilities,
+      thinking on your feet to find creative solutions to problems.
+    </p>
+  </div>
+</section>
+
+<section class="container centered">
+  <article class="campaign">
+    <%= render Content::PhotoQuoteListComponent.new(
+      [
+        {
+          image: "media/images/content/blog/thumbnails/stories-zainab.jpg",
+          heading: "A dynamic work environment",
+          text: "You encounter problems, as you would in any career, but you have the support of colleagues to try new approaches and techniques until you find the right solution.",
+          accreditation: link_to("Read how Zainab develops with the help of her colleagues", page_path("blog/school-experience-helped-me-decide-to-switch")),
+        },
+        {
+          image: "media/images/content/blog/thumbnails/stories-gavin.jpg",
+          heading: "Challenge your own thinking",
+          text: "The classroom changes by the hour and working with different age ranges and abilities means you have to keep a fresh outlook.",
+          accreditation: link_to("Read how Gavin adapts his teaching to create a positive learning environment", page_path("blog/no-two-days-are-the-same")),
```

Comment:
@ethax-ross is it possible to add a full stop after the link here? (can't quite work out how to do it)